elastic/kibana
# fix(slo): Access Cases from SLO details page

Resolves https://github.com/elastic/kibana/issues/198564

### 🐈 Summary

This PR fixes the missing plugin dependency for opening Cases from the Alerts tab under SLO Details page.
I've decided to add it as an optional dependency to avoid blocking the SLO plugin in case it was not enable, since this is used only for that particular feature.


### 🧬 Testing

1. Create an SLO with high objective, let the alerts fire. 
2. Go to the Alerts tab under the SLO Details page
3. Click on the active alert > Add to existing case or Add to new case
4. Assert no error is shown and flyout/popover loads correctly
